### PR TITLE
Standalone component

### PR DESCRIPTION
**Pull request by jedwards4b**
standalone component tests do not produce a cpl log file and so looking there for test completion of these components is a mistake.  This change will cause case_run.py to look in the component log for the completion stamp and will cause memory and performance baseline comparisons to be skipped since memory and performance data is not collected.

Test suite: cesm2.3.alpha17b
Test baseline:
Test namelist changes:
Test status: bit for bit,

Fixes 

User interface changes?:

Update gh-pages html (Y/N)?:

### CODE REVIEW

File: CIME/case/case_run.py [hunk 1]
```diff
@@ -322,9 +318,11 @@ def _post_run_check(case, lid):
         cpl_logs = [os.path.join(rundir, file_prefix + ".log." + lid)]
 
     cpl_logfile = cpl_logs[0]
-
     # find the last model.log and cpl.log
     model_logfile = os.path.join(rundir, model + ".log." + lid)
+    print(
```
Comment (jgfouca) [inline CIME/case/case_run.py:323]: Do we want to keep this print statement? If so, maybe it should be a logger call?

**Review by fischer-ncar — APPROVED** [review]
Looks good to me, except the print statement James pointed out.

File: CIME/case/case_run.py (hunk 1 above)
Comment (jedwards4b) [reply CIME/case/case_run.py]: Thanks for pointing this out.  I have removed it along with another I found.

**Review by jgfouca — APPROVED** [review]

**Comment by jedwards4b** [comment]
@jasonb5 Can I assume this is an expected failure until #4563 is merged?

**Comment by jasonb5** [comment]
Yes these failures are expected and fixed in #4563.

File: CIME/case/case_run.py [hunk 2]
```diff
@@ -292,16 +292,12 @@ def _post_run_check(case, lid):
...
-    model = case.get_value("MODEL")
     driver = case.get_value("COMP_INTERFACE")
-    model = get_model()
 
-    fv3_standalone = False
+    comp_standalone, model = is_comp_standalone(case)
 
-    if "CPL" not in case.get_values("COMP_CLASSES"):
-        fv3_standalone = True
     if driver == "nuopc":
```
Comment (jedwards4b) [inline CIME/case/case_run.py:299]: @rljacob perhaps moving the line 297 to after 299 will solve the problem?

Comment (rljacob) [reply CIME/case/case_run.py:299]: I can try that.

**Pull request merged**